internal/server/iigo.go

```diff
@@ -30,8 +30,8 @@ func (s *SOMASServer) updateIIGOTurnHistory(clientID shared.ClientID, pairs []ru
 }
 
 func (s *SOMASServer) runIIGOTax() error {
-	s.logf("start runIIGOTax")
```

Comment:
@preetl Please just verify that this is what you had in mind i.t.o. the CP contributions integrated into tax requests. Is there anything else that should be added for CP contributions?

Reply:
Looks great! Thanks for making the changes :)